### ISSUE TITLE
Archive Old/Unused Files

### ISSUE THREAD

**Issue by cmccormick1** [rafia37/Evolution-of-Galaxies#15]
Create a separate folder to store the unused files to better organize the repository.

Unused Files:
- Composite Spectra Grid.ipynb
- binning_mag_mass.py
- composite_spectra.pdf
- galaxy_distribution.pdf
- hgamma_to_hbeta.py
- mass_distribution.PNG
- stack_check.py

**Comment by cmccormick1** [comment]
Waiting on review to merge into master. @astrochun
